R/hub_validations_methods.R

```diff
@@ -17,17 +17,18 @@ print.hub_validations <- function(x, ...) {
       ),
       dplyr::case_when(
         purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_success")) ~ "v",
-        purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_failure")) ~ "!",
+        purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_failure")) ~ "x",
         purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_exec_warn")) ~ "!",
-        purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_error")) ~ "x",
-        purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_exec_error")) ~ "x",
+        purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_error")) ~ "circle_cross",
+        purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_exec_error")) ~ "lower_block_8",
         purrr::map_lgl(x, ~ rlang::inherits_any(.x, "check_info")) ~ "i",
         TRUE ~ "*"
       )
     )
   }
-
-  octolog::octo_inform(msg)
+  cli::cli_div(class = "hub_validations", theme = hub_validation_theme)
+  cli::cli_inform(msg)
+  cli::cli_end()
```

Comment:
I think this was a good move. If we really want to have these messages back, it's not terribly difficult to roll our own and we can ensure they still show up on the pkgdown site by checking for the `IN_PKGDOWN` envvar.